test/fixtures/xapi_v3_tracks.json

```diff
@@ -1 +1,142 @@
-{"tracks":[{"id":"animal","language":"Animal","repository":"https://github.com/exercism/xanimal","todo":["apple","banana","cherry","hello-world","one","three","two"],"active":true,"implemented":true,"docs":{"about":"","installation":"","tests":"","learning":"","resources":""},"problems":[{"slug":"dog","name":"Dog","blurb":"This is dog."}]},{"id":"fake","language":"Fake","repository":"https://github.com/exercism/xfake","todo":["apple","banana","cherry","dog"],"active":true,"implemented":true,"docs":{"about":"Language Information\n","installation":"Installing\n","tests":"Running\n","learning":"Learning Fake!\n","resources":"Resources\n"},"problems":[{"slug":"hello-world","name":"Hello World","blurb":"Oh, hello."},{"slug":"one","name":"One","blurb":"This is one."},{"slug":"two","name":"Two","blurb":"This is two."},{"slug":"three","name":"Three","blurb":"This is three."}]},{"id":"fruit","language":"Fruit","repository":"https://example.com/exercism/xfruit","todo":["dog","hello-world","one","three","two"],"active":true,"implemented":true,"docs":{"about":"","installation":"","tests":"","learning":"","resources":""},"problems":[{"slug":"apple","name":"Apple","blurb":"This is apple."},{"slug":"banana","name":"Banana","blurb":"This is banana."},{"slug":"cherry","name":"Cherry","blurb":"This is cherry."}]},{"id":"jewels","language":"Fancy Stones","repository":"https://github.com/exercism/xjewels","todo":["apple","banana","cherry","dog","hello-world","one","three","two"],"active":false,"implemented":false,"docs":{"about":"","installation":"","tests":"","learning":"","resources":""},"problems":[]}]}
+{
```

Comment:
Go ahead and rebase onto master. I've redone the api fixtures and they're all readable.